app/views/users/index.html.erb

```diff
@@ -1 +1,28 @@
-<h1>All users</h1>
\ No newline at end of file
+<h1>All users</h1>
+
+<ul class="users">
+  <% @users.each do |user| %>
+    <li class="flex">
+      <div class="borderb">
+        <img src="<%= asset_path("user-placeholder.png") %>" alt="<%= user.name%>" width=100px >
+      </div>
```

Comment:
- [ ] According to the project requirement, you're required to render the user's photo fetched for a particular. Kindly display 3 data for a user, `user.photo, user.name, and user.posts_counter`. Currently, you're displaying a static image from the assets. Kindly work on this 👍🏽

Reply:
I have worked on it a lot but still struggling to be successful. I have seen other students' projects who have got approval with the same avatar. All code reviewers' marked this as optional requirement. So please let me submit the project.

Reply:
https://github.com/DilsherB/ror-blog-app/assets/41000844/bf00bb55-f78e-42aa-8e43-590647284a86